style/index.less

```diff
@@ -37,13 +37,16 @@
             }
 
             li {
-                box-sizing: border-box;
-                width: 100%;
-                padding: 15px 6px;
                 border-bottom: 1px solid #eee;
+            }
+
+            a {
+                display: block;
+                padding: 15px 6px;
                 line-height: 1.27;
-                font-weight: 500;
-                cursor: pointer;
+				font-weight: 500;
+				color: inherit;
+				transition: all 160ms ease-out;
```

Comment:
ah, sorry for the tab indents. I thought I had converted them all to spaces but must've forgotten these ones. 

adding a `.prettierrc` file would help with contributions 💡

Reply:
thanks, i will fix it, no worries.
yep, should be add standard prettier(fmt)/lint, since just me in this project, currently i am focusing on the new compiler written in Rust.